pyuvdata/miriad.py

```diff
@@ -622,6 +677,54 @@ def write_miriad(self, filepath, run_check=True, check_extra=True,
             uv.add_var('diameter', 'd')
             uv['diameter'] = self.antenna_diameters
 
+        # other extra keywords
+        # set up dictionaries to map common python types to miriad types
+        # NB: arrays/lists/dicts could potentially be written as strings or 1D
+        # vectors.  This is not supported at present!
+        # NB: complex numbers *should* be supportable, but are not currently
+        # supported due to unexplained errors
```

Comment:
Could we be a little more specific about the errors? Maybe `due to errors in aipy.miriad and/or its underlying libraries`. Just to clarify we're not being lazy about debugging our own code.